crates/api_crud/src/comment/delete.rs

```diff
@@ -32,6 +33,8 @@ impl PerformCrud for DeleteComment {
     let data: &DeleteComment = self;
     let local_user_view =
       get_local_user_view_from_jwt(&data.auth, context.pool(), context.secret()).await?;
+    let local_site = LocalSite::read(context.pool()).await?;
+    check_user_approved(&local_user_view, &local_site)?;
```

Comment:
You don't want unapproved users to be able to delete their comments? I could go either way I spose, I just want to make sure that's intentional. And don't forget this will need a front-end check too.